experimental/dds/tree2/src/class-tree/schemaFactory.ts

```diff
@@ -323,28 +343,44 @@ export class SchemaFactory<TScope extends string, TName extends number | string
 						fullName as TName,
 						nameOrAllowedTypes as T,
 						false,
+						true,
 					) as TreeNodeSchema,
 			) as TreeNodeSchemaClass<
 				`${TScope}.${string}`,
 				NodeKind.Map,
 				TreeMapNode<T>,
-				ReadonlyMap<string, InsertableTreeNodeFromImplicitAllowedTypes<T>>
+				ReadonlyMap<string, InsertableTreeNodeFromImplicitAllowedTypes<T>>,
+				true
 			>;
 		}
-		return this.namedMap(nameOrAllowedTypes as TName, allowedTypes, true);
+		return this.namedMap(nameOrAllowedTypes as TName, allowedTypes, true, true);
 	}
 
-	private namedMap<Name extends TName | string, const T extends ImplicitAllowedTypes>(
+	/**
+	 * See note on `namedList`.
```

Comment:
Nit (just in case this class is ever exported):
```suggestion
	 * @privateRemarks See note on `namedList`.
```

Reply:
This is a package public API which should be private, so making its only doc saying why its public private seems like a bad idea.

Reply:
Oh, I misread the situation. Since this is user-facing, it might be better to directly link to the referenced member (if we can), or duplicate the docs :/

Reply:
Or something like :
```typescript
	 * Define a {@link TreeNodeSchema} for a {@link (TreeMapNode:interface)}.
	 *
	 * @param name - Unique identifier for this schema within this factory's scope.
	 *
	 * @remarks See remarks on {@link SchemaFactory.namedlist}.
```